upload/admin/view/template/extension/shipping/zasilkovna_add_vendor.twig

```diff
@@ -0,0 +1,195 @@
+{{ header }}{{ column_left }}
+<div id="content">
+    <div class="page-header">
+        <div class="container-fluid">
+            <h1>{{ text_carriers }}</h1>
+            <ul class="breadcrumb">
+                {% for breadcrumb in breadcrumbs %}
+                    <li><a href="{{ breadcrumb.href }}">{{ breadcrumb.text }}</a></li>
+                {% endfor %}
+            </ul>
+            <div class="pull-right" style="margin-top: -5px;">
+                <button type="submit" form="form-add-vendor" data-toggle="tooltip" title="{{ button_save }}" class="btn btn-primary">
+                    <i class="fa fa-save"></i>
+                </button>
+                <a href="{{ action_back }}" data-toggle="tooltip" title="{{ text_cancel }}" class="btn btn-default"><i class="fa fa-reply"></i> </a>
+            </div>
+        </div>
+    </div>
+    <div class="container-fluid">
+        {% if flashMessage %}
+            <div class="alert {{ flashMessage.class }} alert-dismissible"><i class="fa {{ flashMessage.icon }}"></i> {{ flashMessage.text }}
+                <button type="button" class="close" data-dismiss="alert">&times;</button>
+            </div>
+        {% endif %}
+
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">
+                    <i class="fa fa-pencil"></i> {{ text_add_carrier }} {{ country_name }}: {{ new_vendor_text }}
+                </h3>
+            </div>
+            <div class="panel-body">
+                <form action="{{ link_form_action }}" method="post" id="form-add-vendor" class="form-horizontal">
+                    <input type="hidden" name="action" value="{{ action }}"/>
+                    <input type="hidden" name="country" value="{{ country }}"/>
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group required">
+                                <label class="col-sm-2 control-label"  for="input-vendor">{{ vendor_label }}</label>
+                                <div class="col-sm-10">
+                                    <select name="vendor" id="input-vendor" class="form-control">
+                                        <option value="-1">{{ text_select_vendor }}</option>
```

Comment:
Pokud není v selectu vybraná hodnota, nastav value na prázdný string a ošetři v PHP.  Použít hodnot "-1" je velice nešťastné a vede to k chybám.